pipeline/plot/unit_characteristic_plot.py

```diff
@@ -708,3 +799,18 @@ def plot_paired_coding_direction(unit_g1, unit_g2, labels=None, time_period=None
 
 def get_m_scale(shank_count):
     return 1350 - 150*shank_count
+
+
+def _get_ccf_xyz_max():
+    xmax, ymax, zmax = ccf.CCFLabel.aggr(ccf.CCF, xmax='max(ccf_x)', ymax='max(ccf_y)', zmax='max(ccf_z)').fetch1(
+        'xmax', 'ymax', 'zmax')
+    locals()['_ccf_xyz_max'] = xmax, ymax, zmax
```

Comment:
a bit unsure on locals(), but doesn't this go out of scope immediately after the function returns?

also, maybe this should be a precomputed value since it will be slow?